src/client/Microsoft.Identity.Client/Internal/Broker/NullBroker.cs

```diff
@@ -80,5 +80,10 @@ public Task<MsalTokenResponse> AcquireTokenByUsernamePasswordAsync(Authenticatio
             _logger.Info("NullBroker - returning null on ROPC request.");
             return Task.FromResult<MsalTokenResponse>(null);
         }
+
+        public Dictionary<string, string> GetSsoPolicyHeaders()
+        {
+            return new Dictionary<string, string>();
```

Comment:
Creating a new empty object each time is not good for perf. See `CollectionHelpers.GetEmptyDictionary` instead.

Reply:
Done